src/cli/commands/force/lightning/lwc/preview.ts

```diff
@@ -211,80 +268,69 @@ export default class Preview extends Setup {
         return Promise.resolve();
     }
 
-    public launchPreview(): Promise<boolean> {
-        const platform = this.flags.platform;
-
-        const defaultDeviceName = CommandLineUtils.platformFlagIsIOS(platform)
-            ? iOSConfig.defaultSimulatorName
-            : androidConfig.defaultEmulatorName;
-
-        const device = CommandLineUtils.resolveFlag(
-            this.flags.target,
-            defaultDeviceName
-        );
-
-        const targetApp = CommandLineUtils.resolveFlag(
-            this.flags.targetapp,
-            PreviewUtils.BROWSER_TARGET_APP
-        );
-
-        const projectDir = CommandLineUtils.resolveFlag(
-            this.flags.projectdir,
-            process.cwd()
-        );
+    public async isLwcServerRunning(): Promise<string> {
+        const port = CommonUtils.getLwcServerPort();
+        if (!port) {
+            return Promise.reject(this.unfulfilledMessage);
+        } else {
+            return Promise.resolve(util.format(this.fulfilledMessage, port));
+        }
+    }
 
-        const configFileName = CommandLineUtils.resolveFlag(
-            this.flags.configfile,
-            ''
-        );
+    public launchPreview(): Promise<boolean> {
+        // At this point all of the inputs/parameters have been verified and parsed so we can just use them.
 
-        const component = this.flags.componentname;
+        this.serverPort = '3333'; // default to port 3333
```

Comment:
This assignment feels like it should happen in the declaration of the property. It's "always" the default value for this property.

Reply:
Actually, hold on that. I'd like to suggest a different pattern in the backing `CommonUtils` functionality.

Reply:
Un-hold. I didn't fully understand the different patterns in play with `CommonUtils.getLwcServerPort()` returning a value or `undefined`. I'd keep the suggestion in the original comment, and maybe consider making the '3333' value a constant in `CommonUtils`, since both classes depend on the default port value, albeit for different reasons.

Reply:
ok